contrib/Dockerfile

```diff
@@ -7,8 +7,11 @@
 FROM ubuntu:20.04
 LABEL maintainer="Dgraph Labs <contact@dgraph.io>"
 
-RUN apt-get update && \
-  apt-get install -y --no-install-recommends \
+RUN export DEBIAN_FRONTEND=noninteractive && \
```

Comment:
I have some concerns here for `DEBIAN_FRONTEND=noninteractive` export using `RUN`. 

- Would it make sense to do this via `ARG` or `ENV`?
- Alternatively the article below highlights some issues with `DEBIAN_FRONTEND=noninteractive` setting, as its a cosmetic fix. Could we remove this altogether as we are using `-y`?

reference: 
- https://bobcares.com/blog/debian_frontendnoninteractive-docker/
- https://askubuntu.com/questions/972516/debian-frontend-environment-variable